src/EventStore.Core.Tests/Services/ElectionsService/ClusterSettingsFactory.cs

```diff
@@ -36,7 +36,9 @@ public class ClusterSettingsFactory {
 				TimeSpan.FromSeconds(10),
 				TimeSpan.FromSeconds(10),
 				TimeSpan.FromSeconds(10),
-				TimeSpan.FromSeconds(10), true, Opts.MaxMemtableSizeDefault, Opts.HashCollisionReadLimitDefault, false,
+				TimeSpan.FromSeconds(10), 
+				TimeSpan.FromSeconds(1800),
```

Comment:
Why use 1800 seconds here but 30 minutes in other places? It would be nice to make this consistent in the most easily understood time unit (probably minutes here).